articles/governance/blueprints/overview.md

```diff
@@ -13,7 +13,7 @@ patterns, and requirements. Azure Blueprints makes it possible for development t
 build and stand up new environments with trust they're building within organizational compliance
 with a set of built-in components -- such as networking -- to speed up development and delivery.
 
-Blueprints are a declarative way to orchestrate the deployment of various resource templates and
+Azure Blueprints are a declarative way to orchestrate the deployment of various resource templates and
```

Comment:
This entry is talking about the 'blueprints' object.  It just happens to be at the start of the sentence, and thus capitalized.  Please revert.